test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.AzureImageCodeGeneratorTest.LabelMapping.cs.approved.txt

```diff
@@ -0,0 +1,48 @@
+﻿// This file was auto-generated by ML.NET Model Builder. 
```

Comment:
May want the normal auto-generated header instead.

Reply:
These are from approval test and only for test usage

Reply:
## Resolved